### DISCUSSION TITLE
How can I use brand colors on icons?

### DISCUSSION THREAD

**Discussion by MincePie** [chakra-ui/chakra-ui#6962] [Q&A] [answered]
I'm trying to use the charka UI button that has an icon embedded. I can color the icon differently from the rest of the button if I use the hex value for the icon color. Why can't I use a brand color on the icon?  

This version produces the result I want:
```
<Button leftIcon={<BiNews color="#a63a50"/>} color='brand.sludge' variant='outline' size="sm" fontWeight="normal">
            Media 
          </Button>
```

This version ignores the color attribute on the icon:
```
<Button leftIcon={<BiNews color='brand.orange'/>} color='brand.sludge' variant='outline' size="sm" fontWeight="normal">
            Media 
          </Button>
```

**Comment by TylerAPfledderer** [comment, marked as answer]
Hey @MincePie !

`BiNews` is not a Chakra icon, so it does not directly accept any of the styling configuration to use the theming from Chakra.

You would need to either:
- Pass the component and color prop to Charka's [Icon component](https://chakra-ui.com/docs/components/icon/usage#some-examples) (`<Icon as={BiNews} color='brand.orange' />`), or...
- Use a descendant selector in the `sx` prop from the `Button`
```tsx
sx={{
  & 'svg': {
    color: 'brand.orange'
  }
}}
```
Personally, the first option is much cleaner. 😄
